pkg/gcv/configs/config.go

```diff
@@ -107,10 +108,12 @@ func asConstraintTemplate(data *UnclassifiedConfig) (*ConstraintTemplate, bool)
 	}
 	generatedKind, err := data.Yaml.GetPath("spec", "crd", "spec", "names", "kind").String()
 	if err != nil {
+		glog.Errorf("Failed to retrieve kind field in %s: %v", data.FilePath, err)
```

Comment:
Set verbosity level to 2 and change to info, it is expected to hit this in the process of determining what type of yaml file it is.
I don't think that these logs will be useful most of the time, mostly when debugging.

const logCategorizationVerboseLevel = 2
...
glog.V(logCategorizationVerboseLevel ).Infof(...)

Reply:
We'll reach this path after we've determined that data.Kind is "ConstraintTemplate". In this case I'd think that the .yaml file is malformed somehow. It claims to be a constraint template but we couldn't determine the CRD kind or get the rego field. I think that'd be unexpected. Thoughts?

Reply:
I think that is a better design, but I think the current code structure has the same story.
I have an idea to refactor the code which I don't think it worth shoving into this change let me make up a change and shoot it along to you.

With that in mind I'm fine with either style of logging (err/info) right now.

Reply:
https://github.com/forseti-security/config-validator/pull/33